lib/stemcell/launcher.rb

```diff
@@ -78,9 +76,12 @@ def initialize(opts={})
       @timeout = 300
       @start_time = Time.new
 
-      AWS.config({
+      aws_configs = {:region => @region}
+      aws_configs.merge!({
         :access_key_id     => @aws_access_key,
-        :secret_access_key => @aws_secret_key})
+        :secret_access_key => @aws_secret_key
+      }) if @aws_access_key && @aws_secret_key
```

Comment:
we only ever set `@aws_access_key` and `@aws_secret_key` in a loop above, where we go through `REQUIRED_OPTIONS` and call `instance_variable_set`. now that these are removed from `REQUIRED_OPTIONS`, they were never set. I fixed this in #73 as a side-effect of refactoring EC2 object creation into a function, but just FYI.